### PR TITLE
cli: workflow status progress columns

### PR DESCRIPTION
**Pull request by dinosk**
* ADD step column to show current running step in workflow,
  and show Job.prettified_cmd in command column.

Connects https://github.com/reanahub/reana-client/issues/125

Signed-off-by: Dinos Kousidis <dinos.kousidis@cern.ch>

### CODE REVIEW

File: reana_client/cli/workflow.py [hunk 1]
```diff
@@ -324,62 +364,15 @@ def workflow_status(ctx, workflow, _filter, output_format,
                                                           access_token)
             verbose_headers = ['id', 'user']
             headers = ['name', 'run_number', 'created',
-                       'status', 'progress', 'command']
+                       'status', 'progress']
```
Comment (diegodelemos) [inline reana_client/cli/workflow.py]: Shall we add progress only if the status response contains progress info? To handle the case in which  [progress is not still there](https://github.com/reanahub/reana-workflow-controller/blob/master/reana_workflow_controller/rest.py#L1100)?

Comment (dinosk) [reply reana_client/cli/workflow.py]: Note this won't raise an error but show 0/0 as the progress fields are initialized to 0 when a row gets added in `Job` table. I also think it shouldn't be shown in this case nonetheless.

File: reana_client/cli/workflow.py [hunk 2]
```diff
@@ -308,6 +308,46 @@ def workflow_start(ctx, workflow, access_token):
...
+                headers.remove('command')
+        data.append(list(map(
+            str,
+            [name,
+             run_number,
+             row['created'],
+             row['status'],
+             '{0}/{1}'.
+             format(
+                 row['progress']['succeeded'],
```
Comment (diegodelemos) [inline reana_client/cli/workflow.py]: Can you reproduce?
```console
$ export REANA_WORKON=$(reana-client create -f reana-yadage.yaml)
$ reana-client status
Workflow status could not be retrieved:
'succeeded'
```

Comment (dinosk) [reply reana_client/cli/workflow.py]: yes, I'm working on it I forgot to set the label

File: reana_client/cli/workflow.py [hunk 3]
```diff
@@ -308,6 +308,51 @@ def workflow_start(ctx, workflow, access_token):
 def workflow_status(ctx, workflow, _filter, output_format,
                     access_token, verbose):
     """Get status of previously created workflow."""
+    def _show_progress(succeeded_jobs, total_jobs):
+        if total_jobs:
+            return '{0}/{1}'.format(succeeded_jobs, total_jobs)
+        else:
+            return '-/-'
+
+    def _get_data_from_row(row, data, headers):
```
Comment (diegodelemos) [inline reana_client/cli/workflow.py]: Could we maybe remove `data` as parameter, and return a new list every time? So down, in `workflow_status` scope, when we call it we are always in control of what goes inside `data` making it easier to debug?

**Pull request merged**